modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/checkpoint/CheckpointWorkflow.java

```diff
@@ -196,6 +199,13 @@ public class CheckpointWorkflow {
         this.checkpointWriteOrder = checkpointWriteOrder;
         this.igniteInstanceName = igniteInstanceName;
         this.checkpointCollectPagesInfoPool = initializeCheckpointPool();
+
+        if (SyncFsUtils.SYNC_FS_ENABLED) {
+            if (SyncFsUtils.isEnabled())
```

Comment:
This looks weird. 'If it's enabled and it's enabled'. Also, `isEnabled()` is not about just being enabled, but about being enabled by the user and being available in the OS.

Can `isEnabled()` be renamed to `isActive()` or something else.

Reply:
Good idea